packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -221,6 +222,26 @@ const getAppealType = (appealCaseData) => {
 	return `${appealCaseData.appealTypeName} appeal`;
 };
 
+/**
+ * @param {object} appealCaseData return object from database call
```

Comment:
Now the types change is merged, this can be

`{import('appeals-service-api).Api.AppealCase}`